mslquad/test/test_hello.cpp

```diff
@@ -0,0 +1,31 @@
+// Bring in package's API
+#include <mslquad/px4_base_controller.h>
+ // Bring in gtest
+#include <gtest/gtest.h>
+ 
+// Declare a test
+TEST(TestSuite, testHello){
+  EXPECT_EQ("hello", "hello");
+}
+
+// Declare another test
+TEST(TestSuite, testSucceed){
+  SUCCEED();
+}
+
+TEST(TestSuite, testAssert){
+  int n = 1;
+  ASSERT_EQ(n, 1);
+}
+
+TEST(TestSuite, testFloat){
+  float a,b;
+  a = 1;
+  b = .5;
+  ASSERT_EQ(a+b, 1.5);
+}
```

Comment:
Should add a new line here between the main and test function.

Reply:
done